WordPress/gutenberg
# Adding label/description to `BlockEditor/DuotoneControl`

## What?
This PR ensures that the main duotone picker found in the image duotone filter panel has an identifying label and relevant description.

<img src="https://github.com/WordPress/gutenberg/assets/159848/9f80dd95-8686-41db-8241-87a84568bab2" alt="Screenshot of image duotone filter panel, with hand-drawn arrows connecting a description and the label 'Apply duotone filter' with a duotone picker" width="337px" />

## Why?
As per #54055, every `DuotonePicker` needs to have an appropriate contextual label.

## How?
An ID is generated and assigned to the node containing the popup description. This is linked to the `DuotonePicker` with an `aria-describedby` props. Additionally, an `aria-label` is applied to the `DuotonePicker` matching the tooltip display value.

The `DuotonePicker` component is updated to accept the additional `aria-describedby` prop. (And, as a bonus, the 'unset' option gets a label.)

## Testing Instructions
* Add an image in the editor
* Open the 'Apply duotone filter' popup
* Confirm that the duotone picker has an `aria-describedby` attribute, and that this points to the description
* Confirm that the duotone picker has an `aria-label` attribute
